docs/content/intro/tutorial01.md

```diff
@@ -108,12 +108,3 @@ Re-run the web server and open the browser back to http://127.0.0.1:5000/. You s
 Keep refreshing the page. You should see new UUIDs being generated every time you refresh the page.
 
 When you’re comfortable with the application, read [part 2 of this tutorial][part2] to learn about pushing our application to GitHub.
-
----
-
-Prev: [Quickstart Guide][part1] `|` Next: [Writing your first CI pipeline, Part 2][part2]
-
-[github]: https://github.com/Azure/brigade
```

Comment:
I think some or all of lines 116-119 need to stay, as long as they are referenced in-line above (line 110 for example)